### DISCUSSION TITLE
Chinese Documents Needed？ 需要中文文档和教程？

### DISCUSSION THREAD

**Discussion by mohanchen** [deepmodeling/abacus-develop#2179] [Ideas]
### Background

In order to further assist more users, especially beginners to DFT, do we need Chinese versions of documents?
为了方便更多用户，特别是DFT的初级使用者，我们需要中文文档吗？

### Describe the solution you'd like

We can try。我们可以试试。

**Comment by scott-5** [comment]
For me, it would certainly be nice to have, but not particularly urgent.

**Comment by QuantumMisaka** [comment]
A simple Chinese document about how to quickly prepare ABACUS input files, run ABACUS and read information from output file can be done at first
